Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -11,141 +11,197 @@
 
 package org.kitodo.config;
 
+import java.net.URI;
+import java.nio.file.Paths;
 import java.util.NoSuchElementException;
+import java.util.Optional;
 
+import org.apache.commons.configuration.AbstractConfiguration;
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.configuration.ConversionException;
 import org.apache.commons.configuration.PropertiesConfiguration;
 import org.apache.commons.configuration.reloading.FileChangedReloadingStrategy;
+import org.apache.commons.io.FilenameUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public class Config {
     private static final Logger logger = LogManager.getLogger(Config.class);
     private static volatile PropertiesConfiguration config;
     private static final String CONFIG_FILE = "kitodo_config.properties";
-    private static final String METADATA_DIRECTORY = "MetadatenVerzeichnis";
-    private static final String CONFIG_DIR = "KonfigurationVerzeichnis";
+    public static final int INT_PARAMETER_NOT_DEFINED_OR_ERRONEOUS = 0;
+
+    public static final String DIR_MODULES = "moduleFolder";
 
     /**
-     * Get Kitodo data directory.
+     * Absolute path to the directory that process directories will be created
+     * in, terminated by a directory separator ("/"). The servlet container must
+     * have write permission to that directory.
+     */
+    private static final String DIR_PROCESSES = "MetadatenVerzeichnis";
```

Comment:
Why some of the names are here and some in Parameters class?

I assume that it is because Paramater class is inside the Core and this one in API, but concept is confusing.